packages/app/src/specs/SpecsList.vue

```diff
@@ -69,20 +69,20 @@
     />
     <div
       v-if="specs.length"
-      class="mb-4 grid grid-cols-7 children:font-medium children:text-gray-800"
+      class="mb-4 grid grid-cols-10 children:font-medium children:text-gray-800"
       :style="`padding-right: ${scrollbarOffset}px`"
     >
       <div
-        class="flex col-span-3 items-center justify-between"
+        class="flex col-span-4 items-center justify-between"
         data-cy="specs-testing-type-header"
       >
         {{ props.gql.currentProject?.currentTestingType === 'component' ?
           t('specPage.componentSpecsHeader') : t('specPage.e2eSpecsHeader') }}
       </div>
-      <div class="flex col-span-2 items-center justify-between">
+      <div class="flex items-center justify-between whitespace-nowrap">
```

Comment:
not sure if there is a better way to do this - this was the only header that was wrapping on smaller screens